<!DOCTYPE HTML>
<html lang="en">
  <head>
    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">

    <title>Jiating Yu</title>

    <meta name="author" content="Jiating Yu">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <link rel="shortcut icon" href="images/favicon/favicon.ico" type="image/x-icon">
    <link rel="stylesheet" type="text/css" href="stylesheet.css">
    
  </head>

  <body>
    <table style="width:100%;max-width:800px;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>
      <tr style="padding:0px">
        <td style="padding:0px">
          <table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>
            <tr style="padding:0px">
              <td style="padding:2.5%;width:63%;vertical-align:middle">
                <p class="name" style="text-align: center;">
                  Jiating Yu (于嘉汀)
                </p>
                <p> I'm currently a assistant professor at <a href="https://www.nuist.edu.cn/"> Nanjing University of Information Science and Technology </a> in China</a>.
				<p> Before joining NUIST, I got my Ph.D. in Operations Research at <a href="http://www.amss.ac.cn/"> Academy of Mathematics and Systems Science, Chinease Academy of Sciences </a>, advised by <a href="http://wulab.ac.cn/">Professor Ling-Yun Wu</a>.
                <p> I received my B.S. in Mathematics and Applied Mathematics at <a href="https://www.xtu.edu.cn/">Xiangtan University</a> in 2014, where I spent four wonderful years of my life.</p>
				</p>
                </p>
                <p style="text-align:center">
                  <a href="mailto:yujiating@amss.ac.cn">Email</a> &nbsp;/&nbsp;
                  <a href="data/JiatingYu_CV.pdf">CV</a> &nbsp;/&nbsp;
                  <a href="https://scholar.google.com/citations?user=cFEHcPsAAAAJ">Scholar</a> &nbsp;/&nbsp;
                  <a href="https://github.com/jiatingyu-amss">Github</a>
                </p>
              </td>
              <td style="padding:2.5%;width:40%;max-width:40%">
                <a href="images/jiating.jpg"><img style="width:100%;max-width:100%;object-fit: cover; border-radius: 50%;" alt="profile photo" src="images/jiating_circle.jpg" class="hoverZoomLink"></a>
              </td>
            </tr>
          </tbody></table>
          <table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>
              <tr>
              <td style="padding:20px;width:100%;vertical-align:middle">
                <h2>Research</h2>
                <p>
                  I'm interested in bioinformatics, computational biology, machine learning and optimization.
                </p>
              </td>
            </tr>
          </tbody></table>
          <table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>





            <tr>
              <td style="padding:20px;width:25%;vertical-align:middle">
                <img src="images/MOLI.jpg" alt="b3do" width="300" style="border-style: none">
              </td>
              <td width="75%" valign="middle">
                <a href="https://doi.org/10.1016/j.physa.2022.127522">
                  <span class="papertitle">Multiple Order Local Information model for link prediction in complex networks</span>
                </a>
                <br>
                <a><strong>Jiating Yu</strong></a>,
                <a>Ling-Yun Wu</a>
                <br>
                <em>Physica A: Statistical Mechanics and its Applications</em>, 2022
                <br>
                <a href="data/MOLI.bib">bibtex</a> /
                <a href="https://github.com/Wu-Lab/MOLI">code</a>
                <p> We proposed a novel method named Link prediction using Multiple Order Local Information (MOLI) that exploits the local information from the neighbors of different distances, with parameter that can be a prior-driven based on prior knowledge, or data-driven by solving an optimization problem on observed networks. MOLI defined a local network diffusion process via random walks on the graph, resulting in better use of network information. We show that MOLI outperforms the other 12 widely used link prediction methods on 15 different types of simulated and real-world networks. We also conclude that there are different patterns of local information utilization for different networks, including social networks, communication networks, biological networks, etc.</p>
              </td>
            </tr>
			
			

            <tr>
              <td style="padding:20px;width:25%;vertical-align:middle">
                <img src="images/NR.jpg" alt="b3do" width="300" style="border-style: none">
              </td>
              <td width="75%" valign="middle">
                <a href="https://doi.org/10.1016/j.physa.2023.128681">
                  <span class="papertitle">Network Refinement: Denoising complex networks for better community detection</span>
                </a>
                <br>
                <a><strong>Jiating Yu</strong></a>,
                <a>Jiacheng Leng</a>,
                <a>Duanchen Sun</a>,
                <a>Ling-Yun Wu</a>
                <br>
                <em>Physica A: Statistical Mechanics and its Applications</em>, 2023
                <br>
                <a href="data/NR.bib">bibtex</a> /
                <a href="https://github.com/Wu-Lab/NR">code</a>
                <p> In this study, we highlighted the necessity of using network denoising as a pre-processing step to improve the performance of community detection. We proposed a novel network denoising method, called Network Refinement (NR), which used a global diffusion process defined by random walk on graph to enhance the self-organization properties of complex networks. NR took a noisy network as input and output a denoised network with clearer community structure by adjusting the edge weights. We have proved that NR can be understood as a degree normalized version of the Katz index, which renders paths with higher intermediate node degrees less important because their information is dispersed through more adjacent edges. We have showed through sufficient numerical experiments that NR significantly improves the clarity of the network’s mesoscale structure, and NR can be applied as a pre-processing step to substantially boost the performance of various community detection algorithms on both simulated networks and real-world networks.</p>
              </td>
            </tr>
			
			
			

            <tr>
              <td style="padding:20px;width:25%;vertical-align:middle">
                <img src="images/SCARP.jpg" alt="b3do" width="300" style="border-style: none">
              </td>
              <td width="75%" valign="middle">
                <a href="https://doi.org/10.1093/bib/bbae093">
                  <span class="papertitle">Incorporating network diffusion and peak location information for better single-cell ATAC-seq data analysis</span>
                </a>
                <br>
                <a><strong>Jiating Yu</strong></a>,
                <a>Jiacheng Leng</a>,
                <a>Zhichao Hou</a>,
                <a>Duanchen Sun</a>,
                <a>Ling-Yun Wu</a>
                <br>
                <em>Briefings in Bioinformatics</em>, 2024
                <br>
                <a href="data/SCARP.bibtex">bibtex</a> /
                <a href="https://github.com/Wu-Lab/SCARP">code</a>
                <p> We proposed a novel network diffusion-based computational method to comprehensively analyze scATAC-seq data, named Single-Cell ATAC-seq Analysis via Network Refinement with Peaks Location Information (SCARP). SCARP formulates the Network Refinement diffusion method under the graph theory framework to aggregate information from different network orders, effectively compensating for missing signals in the scATAC-seq data. By incorporating distance information between adjacent peaks on the genome, SCARP also contributes to depicting the co-accessibility of peaks. These two innovations empower SCARP to obtain lower-dimensional representations for both cells and peaks more effectively. We have demonstrated through sufficient experiments that SCARP facilitated superior analyses of scATAC-seq data.
                </p>
              </td>
            </tr>


			

            <tr>
              <td style="padding:20px;width:25%;vertical-align:middle">
                <img src="images/RENDOR.jpg" alt="b3do" width="300" style="border-style: none">
              </td>
              <td width="75%" valign="middle">
                <a href="https://doi.org/10.1093/bioinformatics/btae435">
                  <span class="papertitle">Reverse network diffusion to remove indirect noise for better inference of gene regulatory networks</span>
                </a>
                <br>
                <a><strong>Jiating Yu</strong></a>,
                <a>Jiacheng Leng</a>,
                <a>Fan Yuan</a>,
                <a>Duanchen Sun</a>,
                <a>Ling-Yun Wu</a>
                <br>
                <em>Bioinformatics</em>, 2024
                <br>
                <a href="data/RENDOR.bibtex">bibtex</a> /
                <a href="https://github.com/Wu-Lab/RENDOR">code</a>
                <p>In this study, we proposed a novel network denoising method named REverse Network Diffusion On Random walks (RENDOR). RENDOR is designed to enhance the accuracy of GRNs afflicted by indirect effects. RENDOR takes noisy networks as input, models higher-order indirect interactions between genes by transitive closure, eliminates false-positive effects using the inverse network diffusion method, and produces refined networks as output. We conducted a comparative assessment of GRN inference accuracy before and after denoising on simulated networks and real GRNs. Our results emphasized that the network derived from RENDOR more accurately and effectively captures gene interactions. This study demonstrates the significance of removing network indirect noise and highlights the effectiveness of the proposed method in enhancing the signal-to-noise ratio of noisy networks.</p>
              </td>
            </tr>
			
			
			
			

            <!-- <tr>
              <td style="padding:20px;width:25%;vertical-align:middle">
                <img src="images/TAPD.jpg" alt="b3do" width="300" style="border-style: none">
              </td>
              <td width="75%" valign="middle">
                <a href="https://drive.google.com/file/d/1YvRx-4hrZoCk-nl6OgVJZlHAqOiN5hWq/view?usp=sharing">
                  <span class="papertitle">Cleaning the USNO-B Catalog Through Automatic Detection of Optical Artifacts</span>
                </a>
                <br>
                <strong>Jonathan T. Barron</strong>, <a href="http://stumm.ca/">Christopher Stumm</a>, <a href="http://cosmo.nyu.edu/hogg/">David W. Hogg</a>, <a href="http://www.astro.princeton.edu/~dstn/">Dustin Lang</a>, <a href="http://cs.nyu.edu/~roweis/">Sam Roweis</a>
                <br>
                <em>The Astronomical Journal</em>, 135, 2008
                <p>We use computer vision techniques to identify and remove diffraction spikes and reflection halos in the USNO-B Catalog.</p>
                <p>In use at <a href="http://www.astrometry.net">Astrometry.net</a></p>
              </td>
            </tr> -->
			
			
			

          </tbody></table>

          
          <table width="100%" align="center" border="0" cellspacing="0" cellpadding="20"><tbody>
            <tr>
              <td>
                <h2>Awards</h2>
              </td>
            </tr>
          </tbody></table>
          <table width="100%" align="center" border="0" cellpadding="20"><tbody>
            
            <tr>
              <td style="padding:20px;width:25%;vertical-align:middle"><img src="images/award.png"></td>
              <td width="75%" valign="center">
                <a>2022, Schlumberger Scholarships, University of Chinese Academy of Sciences.</a>
                <br>
                <a>2020, Merit students, University of Chinese Academy of Sciences.</a>
                <br>
                <a>2018, Xiangtan University 2018 Outstanding Graduates, Xiangtan University.</a>
                <br>
                <a>2018, Xiangtan Outstanding Graduates, Xiangtan University.</a>
                <br>
                <a>2017, National Inspiration Scholarship, Xiangtan University.</a>
                <br>
                <a>2016, Outstanding Student Cadre, Xiangtan University.</a>
                <br>
                <a>2016, National Scholarships, Xiangtan University.</a>
                <br>
                <a>2016, Scholarships in Basic Subjects, Xiangtan University.</a>
                <br>
                <a>2016, Advanced Individual, Xiangtan University.</a>
                <br>
                <a>2016, Second Prize of Mathematics Competition for College Students in Hunan Province (Mathematics Category), Xiangtan University.</a>
                <br>
                <a>2016, Third Prize of the Eighth National University Student Mathematics Competition (Mathematics Major), Xiangtan University.</a>
                <br>
                <a>2015, Class A Scholarships, Xiangtan University.</a>
                <br>
                <a>2015, Xiangjiang Scholarships, Xiangtan University.</a>
              </td>
            </tr>
            
            
            
          </tbody></table>
          <table style="width:100%;border:0px;border-spacing:0px;border-collapse:separate;margin-right:auto;margin-left:auto;"><tbody>
            <tr>
              <td style="padding:0px">
                <br>
                <p style="text-align:right;font-size:small;">
                (*^▽^*)
                </p>
              </td>
            </tr>
          </tbody></table>
        </td>
      </tr>
    </table>
  </body>
</html>
